src/views/catalog/templatescatalog/TemplatesCatalog.tsx

```diff
@@ -25,7 +25,7 @@ const TemplatesCatalog: React.FC<RouteComponentProps<{ ns: string }>> = ({
   },
 }) => {
   const { t } = useKubevirtTranslation();
-  const [isAdmin, isAdminLoaded] = useIsAdmin();
+  const isAdmin = useIsAdmin();
```

Comment:
```suggestion
  const canListNS =  useFlag('CAN_LIST_NS');
```

IMHO using `CAN_LIST_NS` is more correct because we don't actually test for admin user, we test for any user that can list namespaces, I also like using the canListNS approach because it's reduces the need for the isAdmin wrapper.

Reply:
btw/ this should actually be two PRs, one for version bumb, and one for using the new hook ...